src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -156,9 +162,12 @@ class JesBackend extends Backend with LazyLogging {
     val engineFunctions = new JesEngineFunctions(GoogleCloudStoragePath(callGcsPath), JesConnection)
 
     // FIXME: Not particularly robust at the moment.
```

Comment:
This doesn't seem to apply to anything nearby. Maybe a general comment on the entire file? In any case it's probably not particularly useful so I'll remove.